controllers/url.go

```diff
@@ -46,6 +46,25 @@ func CreateTinyURL(ctx *gin.Context, db *bun.DB) {
 	}
 
 	ctx.JSON(http.StatusOK, gin.H{
-		"message": "Tiny URL created successfully",
+		"message":   "Tiny URL created successfully",
+		"short_url": body.ShortUrl,
 	})
 }
+
+func RedirectShortURL(ctx *gin.Context, db *bun.DB) {
```

Comment:
The function first retrieves the shortened URL from the context using ctx.Param("shortURL"). Then, it attempts to fetch the corresponding original URL from the database using the bun.DB object. If the short URL is not found, it returns a JSON response with an error message. Otherwise, it redirects the user to the original URL using ctx.Redirect

Reply:
I Have done the way you have commented over, does I have done anything wrong then please suggest me I'll do that